python/runDataMC_ttbarAC.py

```diff
@@ -26,16 +26,47 @@
 import Analysis.CyMiniAna.hepPlotter.hepPlotterLabels as hpl
 import Analysis.CyMiniAna.util as util
 
+selection = 'mujets'
```

Comment:
This section needs to be cleaned up.  Use the metadata text file?

Reply:
Future update that will include "official" list of samples (for analysis use) and scripts to make "official" plots from those samples